docs/fundamentals/code-analysis/code-quality-rule-options.md

```diff
@@ -73,23 +73,31 @@ The syntax for configuring an option for a *specific* rule is as follows:
 
 This section lists some of the available options. To see the full list of available options, see [Analyzer configuration](https://github.com/dotnet/roslyn-analyzers/blob/main/docs/Analyzer%20Configuration.md).
 
-- [api_surface](#api_surface)
-- [exclude_async_void_methods](#exclude_async_void_methods)
-- [exclude_single_letter_type_parameters](#exclude_single_letter_type_parameters)
-- [output_kind](#output_kind)
-- [required_modifiers](#required_modifiers)
-- [exclude_extension_method_this_parameter](#exclude_extension_method_this_parameter)
-- [null_check_validation_methods](#null_check_validation_methods)
-- [additional_string_formatting_methods](#additional_string_formatting_methods)
-- [excluded_type_names_with_derived_types](#excluded_type_names_with_derived_types)
-- [excluded_symbol_names](#excluded_symbol_names)
-- [disallowed_symbol_names](#disallowed_symbol_names)
+- [Code quality rule configuration options](#code-quality-rule-configuration-options)
+  - [Option scopes](#option-scopes)
```

Comment:
These aren't in this `## Options` section. Did you use a tool to generate this list?

Reply:
I have no idea where this stuff comes from. I'll undo those changes.

Reply:
Turns out this is from VS Code. If I edit anything in this file, it insists on replacing this set of lines when I save. Very strange.